release-notes.md

```diff
@@ -1,4 +1,14 @@
 # Ursa-21.10.0
+
+## Enable ScheduleEventNotificationPlugin
+- Issue Type: Task
+- Issue ID: PSVAMB-69071
+#### Deployment ####
+- Generate Clients
+
+### Deployment Scripts ###
+    php deployment/base/scripts/installPlugins.php
+
```

Comment:
What about the actual ini change? I.e adding the plugin name to plugins.ini

Reply:
I'll add instructions to make sure it's enabled. However in most cases the config files are in a different repo altogether